docs/reference/commandline/volume_ls.md

```diff
@@ -33,9 +34,51 @@ Example output:
     local               rose
     local               tyler
 
+## Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
+than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
+
+* dangling (boolean - true or false, 0 or 1)
+* driver (a volume driver's name)
+* name (a volume's name)
+
+### dangling
+
+The `dangling` filter matches on all volumes not referenced by any containrs
+
+    $ docker run -d  -v tyler:/tmpwork  busybox
+    f86a7dd02898067079c99ceacd810149060a70528eff3754d0b0f1a93bd0af18
+    $ docker volume ls -f dangling=true
+    DRIVER              VOLUME NAME
+    local               rose
+
+### driver
+
+The `driver` filter matches on all or part of a volume's driver name.
+
+The following filter matches all volumes with a driver name containing the `local` string.
+
+    $ docker volume ls -f driver=local
+    DRIVER              VOLUME NAME
+    local               rose
+    local               tyler
+
+### name
+
+The `name` filter matches on all or part of a volume's name.
+
+The following filter matches all volumes with a name containing the `rose` string.
+
+    $ docker volume ls -f name=rose
+    DRIVER              VOLUME NAME
+    local               rose
```

Comment:
perhaps we should change the volume name to (e.g.) `rosemary` here to show that it actually includes "partial" matches as well